src/sagemaker/image_utils.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional, Tuple
+
+from sagemaker.image_uris import config_for_framework, retrieve
+
+
+def get_latest_container_image(framework: str,
```

Comment:
Does this work with all framework type or just subset? Can it work if user wants to get latest for distributed training as well or this meant to just be like very simple for few framework

Reply:
Yeah this needs some rework since some of them require instance type and python versions . Will update that . 

Can you point me to some config jsons that are specifically for distributed training ?